api/src/routes/router.ts

```diff
@@ -95,10 +96,12 @@ verificationRouter.post(
 	revokeVerifiableCredential
 );
 
+const subscriptionPool = new SubscriptionPool(streamsNode);
+subscriptionPool.startInterval();
```

Comment:
do you also call a stopInterval function? 
for example this can be done as part of `try/catch/finally` wrapper

Reply:
No I don't call it, it shall run the whole time the api is running to remove obsolete objects from the pool. Should I remove stopInterval then?

Reply:
I would add it into the `finally` block to intercept abnormal terminations and avoid memory pollution

Reply:
however this is optional and can be skipped